js/home.js

```diff
@@ -0,0 +1,10 @@
+var $a = document.getElementById("nav");
+window.onscroll = function() {scrollFunction()};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['scrollFunction' was used before it was defined.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)